include/cute/atom/copy_traits_xe_2d.hpp

```diff
@@ -774,6 +784,7 @@ make_block_2d_copy_A(CopyOp                   const& op,    // Copy operation
                      TiledMMA                 const& mma,   // TiledMMA instance
                      Tensor<GEngine, GLayout> const& gmem)  // Global tensor
 {
+  static_assert(is_xe_atom_v<CopyOp>, "Use new XE atoms: XE_LOAD_2D or XE_LOAD_2D_TRANSPOSE");
```

Comment:
Let's not call this "new" as it's only new for now :) Also, I wouldn't suggest a particular kind of block 2D atom, as regular/VNNI/transpose can all be used for the A matrix, in different situations.
```suggestion
  static_assert(is_xe_atom_v<CopyOp>, "Expected a block 2D atom");
```